### PR TITLE
CI: check docs

### PR DESCRIPTION
**Pull request by MarcoIeni**
working example: https://github.com/LukeMathWalker/wiremock-rs/pull/66

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -194,3 +194,20 @@ jobs:
...
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          profile: minimal
+          override: true
+      - name: Check documentation
+        env:
+          RUSTDOCFLAGS: -D warnings
+        run: cargo doc --no-deps --document-private-items
```
Comment (BurntSushi) [inline .github/workflows/ci.yml:213]: This should probably have `--all` in it, right?

Comment (MarcoIeni) [reply .github/workflows/ci.yml]: Yes, I also added `--all-features`

**Comment by MarcoIeni** [comment]
I have to look into this issue. By the way, I just noticed the `--all` flag is deprecated. Should we use `--workspace` instead?

```
$ cargo doc --help       
cargo-doc 
Build a package's documentation

USAGE:
    cargo doc [OPTIONS]

OPTIONS:
[...]
    -p, --package <SPEC>...          Package to document
        --all                        Alias for --workspace (deprecated)
        --workspace                  Document all packages in the workspace
[...]
```

**Comment by BurntSushi** [comment]
No, not `--all-features`. Just `--all`. Errmm, well, I guess `--workspace`. I didn't know they deprecated `--all`. Sigh.

**Pull request closed without merging**